src/main/java/com/faforever/client/player/PlayerServiceImpl.java

```diff
@@ -147,7 +147,14 @@ private void updateGamePlayers(List<String> players, Game game) {
   }
 
   @Override
+  public boolean isOnline(String username) {
+    //attention if called while loading
```

Comment:
Not sure what this comment tells us. Either replace it with code (e.g. throw exception if a condition isn't met) or remove it